cli/operator/node.go

```diff
@@ -457,10 +461,10 @@ var StartNodeCmd = &cobra.Command{
 			})
 		}
 
-		if cfg.SSVOptions.ValidatorOptions.Exporter {
-			retain := cfg.SSVOptions.ValidatorOptions.ExporterRetainSlots
+		if cfg.ExporterOptions.Enabled && cfg.ExporterOptions.Mode == "" {
```

Comment:
I see the default is "standard" so will `cfg.ExporterOptions.Mode == ""` ever be true? 
is this a case where the user specifies "" for some reason?